docs/settings.md

```diff
@@ -18,6 +18,11 @@ The DocumentParser class used for .docx files. Defaults to `DocxParser`.
 Whether Wagtail Content Import should check the file content as well as the hash when determining duplicate images.
 Defaults to `False`.
 
+#### `WAGTAILCONTENTIMPORT_CONVERT_EXTERNAL_LINKS`
```

Comment:
Where does this get used?

Reply:
It's used over here: https://github.com/torchbox/wagtail-content-import/pull/47/files#diff-64e68453787903389a302250d1fc60f8dc2fa25a315c12c731ba0bd61318ec54R99

Reply:
Oh, there it is!

I could only see the tests earlier! Not sure how I managed to miss that file